.pre-commit-hooks.yaml

```diff
@@ -1,3 +1,9 @@
+-   id: python-check-blanket-noqa
+    name: Check blanket noqa
+    description: 'Enforce that `noqa` annotation always occurs with specific codes to avoid blanket noqas'
```

Comment:
I'd say "Enforce that `noqa` annotation*s* always occur with specific codes"

Reply:
Fixed.